### ISSUE TITLE
Bug report - [Package Broken]

### ISSUE THREAD

**Issue by 0xrobiul** [CypherpunkArmory/UserLAnd#1640]
# Description 

Packages Are Broken, So It's Failed To Upgrade OS!

# Steps to reproduce the behavior:
1. Just Try `apt upgrade` with root permission.

# Device Information
  
Android Version: 11.  
UserLAnd Version 3.0.0
Whether the device is non-rooted.  
Whether the device is running Kali (Terminal)

**Comment by antonialoytorrens** [comment]
Have you run `apt update`?

**Comment by 0xrobiul** [comment]
Yep, To Validate The Bug Just Try `apt upgrade` After `apt update`! 

For Some Reason I Can't Provide Screenshot Right Now But The Error Code Is Something Like Below:

```
ionice: ioprio_set failed: Operation not permitted
chrt: failed to set pid 15365's policy: Operation not permitted
cp: cannot create regular file '/usr/support/libdisableselinux.so': No such file or directory

FATAL ERROR:
cp --no-dereference --preserve=all --reflink=auto --sparse=always /support/libdisableselinux.so /usr/support/libdisableselinux.so: rc=1

You can try correcting the errors reported and running again
/usr/lib/usrmerge/convert-usrmerge until it will complete without errors.
Do not install or update other Debian packages until the program
has been run successfully.

E: usrmerge failed.
dpkg: error processing package usrmerge (--configure):
 installed usrmerge package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 usrmerge
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Hope You'll Like To Check It Out!!

**Comment by 0xrobiul** [comment]
Here's Screenshot!
![1668349130061](https://user-images.githubusercontent.com/100078094/201526609-619d526a-08b6-471f-8a85-16e9af7de17d.jpg)

**Comment by loreb** [comment]
I ran into the same issue after I noticed I was still on debian/oldstable; there simply needs to be a directory called /usr/support, so I ran `mkdir /usr/support && chmod --reference=/support /usr/support` (not sure if the chmod is needed but it shouldn't hurt), then apt upgrade worked and I'm now on stable.

I needed to explicitly install passwd afterwards (because I ran apt **upgrade** instead of dist-upgrade); I had a (harmless) warning about static linking for a package I had installed; and for some reason the keyboard appears to sometimes ignore keypresses? Whatever, the point is that you just need to create a directory.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
